client/tracks/index.js

```diff
@@ -31,10 +31,10 @@ function recordEvent( eventName, eventProperties ) {
 }
 
 /**
- * Records events from buyers.
+ * Records events from buyers (aka shoppers).
  *
- * @param {string}  eventName       Name of the event.
- * @param {Object?} eventProperties Event properties.
+ * @param {string}  eventName       	Name of the event.
```

Comment:
Too many spaces making the description does not align well with the next line. Looks like there is a `tab` there.

Here's how I see it:
![Screenshot 2023-05-02 at 22 51 49](https://user-images.githubusercontent.com/73803630/235718734-5b0448c8-d6c4-4a37-a0f9-064ea841885c.png)

Reply:
Good catch, I'll fix that using spaces (and check the other change). Looks like I need to configure my editor.

Reply:
Fixed in 6ec9c3735e749f1fbd15a667917c26a992f96d84 (and used the smallest number of spaces needed).